app/src/test/java/org/mozilla/focus/engine/CustomContentRequestInterceptorTest.kt

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.engine
+
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.concept.engine.request.RequestInterceptor
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+@RunWith(RobolectricTestRunner::class)
+class CustomContentRequestInterceptorTest {
+    @Test
+    fun `Interceptor should return content for firefox-home`() {
+        val result = testInterceptor("firefox:home")
+
+        assertNotNull(result)
+        assertEquals("<html></html>", result!!.data)
+        assertEquals("text/html", result.mimeType)
+        assertEquals("UTF-8", result.encoding)
+    }
+
+    @Test
+    fun `Interceptor should return content for firefox-about`() {
+        val result = testInterceptor("firefox:about")
+
+        assertNotNull(result)
+        assertTrue(result!!.data.isNotEmpty())
+        assertEquals("text/html", result.mimeType)
+        assertEquals("UTF-8", result.encoding)
+    }
+
```

Comment:
Optional nit: Might be worth adding a small test that asserts that "firefox:home" and "firefox:about" return values are not equal, to avoid false positives.

Reply:
That's a great idea. I'll add that.